docs/user/admin/integrations/supported-integrations.md

```diff
@@ -1,26 +1,24 @@
-
-Atomist receives events from and performs operations on many systems,
+Atomist isn't designed to replace your CI service; rather, the goal of the SDM is to enhance all of your existing pipelines and policies through additional automation. It does so by receives and reacting to events from many systems,
```

Comment:
> ~Atomist isn't designed to replace your CI service; rather,~
I suggest striking that phrase. The underlying idea is that you don't have to replace your CI in order to use Atomist.